multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -64,10 +64,17 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 # CIDR for Kubernetes VPC. If vpcId is specified, must match the CIDR of existing vpc.
 # vpcCIDR: "10.0.0.0/16"
 
-# CIDR for Kubernetes subnet
+# CIDR for Kubernetes subnet when placing nodes in a single availability zone (not highly-available) Leave commented out for multi availability zone setting and use the below `subnets` section instead.
 # instanceCIDR: "10.0.0.0/24"
 
-# IP Address for controller in Kubernetes subnet
+# Kubernetes subnets with their CIDRs and availability zones. Differentiating availability zone for 2 or more subnets result in high-availability (failures of a single availability zone won't result in immediate downtimes)
```

Comment:
Should we make any notes about EBS volumes being zone specific?  From what I can tell, if you're safe unless you depend on an EBS volume in the AZ that goes down.